docker/Dockerfile.graph

```diff
@@ -1,24 +0,0 @@
-FROM vesoft/nebula-dev:centos7 as builder
```

Comment:
Don't delete this file now. Some documents still reference the `vesoft/nebula-graph` image. And this dockerfile is used to build that image.

Reply:
ok